posts/2025-08-26-25.0.0.9-beta.adoc

```diff
@@ -0,0 +1,182 @@
+//
+// Throughout the doc, replace 25.0.0.9-beta with the version number of Open Liberty, eg: 22.0.0.2-beta
+// // // // // // // //
+
+* <<fips, FIPS 140-3 support on IBM Semeru>>
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/32410
+// Contact/Reviewer: jimmy1wu
+// // // // // // // // 
+[#fips]
+== FIPS 140-3 support on IBM Semeru
+
+FIPS 140-3 is the latest version of the FIPS 140 standard, which provides a set of guidelines for cryptographic modules to ensure their security and integrity. Support for FIPS 140-3 was first introduced in Liberty with IBM JDK 8 in version link:https://openliberty.io/blog/2025/03/25/25.0.0.3.html[25.0.0.3]. In this release, Open Liberty supports the FIPS 140-3 standard when using link:https://developer.ibm.com/languages/semeru-runtimes[IBM Semeru Runtimes] versions 11.0.27, 17.0.15, 21.0.7, or higher. 
+
+FIPS 140-3 is enabled when:
+- security features utilizing crypto algorithms are used 
+- FIPS 140-3 Semeru jvm properties are set.
+
+If you installed the IBM Semeru Runtimes, you can enable FIPS 140-3 by setting the following system properties, for example in the `jvm.options` file:
+
+----
+-Dcom.ibm.ws.beta.edition=true
+-Dsemeru.fips=true
+-Dsemeru.customprofile=OpenJCEPlusFIPS.FIPS140-3
+----
+
+The `OpenJCEPlusFIPS.FIPS140-3` profile is the strict security profile defined by the IBM Semeru Runtime, as outlined in the link:https://www.ibm.com/support/pages/fips-140-3-cryptography-ibm-semeru-runtimes#:~:text=FIPS140%2D3%20restricted%20security%20mode%20profile[official documentation]. This profile enforces the use of only FIPS 140-3 approved cryptographic algorithms, along with non-cryptographic algorithms that are either explicitly permitted or unrelated to the FIPS 140-3 specification.
+
+Depending on your specific use case, you may need to apply additional constraints by link:https://www.ibm.com/support/pages/fips-140-3-cryptography-ibm-semeru-runtimes#:~:text=Creating%20extensions%20of%20profiles[creating an extension of the profile]. For instance, if your application uses PKCS12 as the TLS certificate keystore and truststore, you’ll need to define a custom profile that includes support for PKCS12.
+
+Refer to this link:https://www.ibm.com/support/pages/fips-140-3-cryptography-ibm-semeru-runtimes#:~:text=PKCS12%20and%20JKS%20Example[example profile] for guidance on how to extend the strict profile to accommodate PKCS12-based configurations.
+
+To support functionality beyond the default `OpenJCEPlusFIPS.FIPS140-3` profile, you must update your custom profile with additional constraints. These constraints should be carefully reviewed to ensure compliance with the FIPS 140-3 specification for your specific use case. 
```

Comment:
Instead of "These constraints should be carefully reviewed to ensure compliance with the FIPS 140-3 specification for your specific use case. " say "Carefully review the constraints to ensure compliance with the FIPS 140-3 specification for your specific use case"